### PR TITLE
Completed max_unpool1d implmentation

### PR DESCRIPTION
**Pull request by mohame54**

### CODE REVIEW

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

File: ivy/data_classes/container/experimental/layers.py [hunk 1]
```diff
@@ -2280,3 +2280,112 @@ def rfftn(
...
+        Returns
+        -------
+        ret
+            The result of the unpooling operation.
+        """
+        return ContainerBase.cont_multi_map_in_function(
+            "max_unpool1d",
+            input,
+            indices=indices,
+            kernel_size=kernel_size,
```
Comment (AnnaTz) [inline ivy/data_classes/container/experimental/layers.py:2339]: You are passing `indices` and `kernel_size` as keyword arguments and this is causing failures.

File: ivy/data_classes/container/experimental/layers.py [hunk 2]
```diff
@@ -2280,3 +2280,112 @@ def rfftn(
...
+            strides=strides,
```
Comment (AnnaTz) [inline ivy/data_classes/container/experimental/layers.py:2340]: Also, you still seem to be committing changes to the `demos` file. Don't know which IDE you are using but you should be able to exclude that folder.

**Review by AnnaTz — CHANGES_REQUESTED** [review]

File: ivy/data_classes/container/experimental/layers.py (hunk 2 above)
Comment (mohame54) [reply ivy/data_classes/container/experimental/layers.py:2340]: I don't know it keeps giving me errors

File: ivy/data_classes/container/experimental/layers.py (hunk 1 above)
Comment (mohame54) [reply ivy/data_classes/container/experimental/layers.py]: yeah, Indeed I am sorry I didn't notice it.

**Comment by AnnaTz** [comment]
Hey @mohame54, there are still errors in your code caught by the CI checks. Please resolve them.

**Pull request closed without merging**